src/com/company/GameField.java

```diff
@@ -0,0 +1,85 @@
+package com.company;
+
+public class GameField {
+
+    private int[][] array;
+    private int squareX;
+    private int squareY;
+    private int result;
+    static int prize = 7;
```

Comment:
1. such fields can be static and final
2. naming convention - `STATIC_FINAL_FIELD`